### PR TITLE
Feature/tao 10138/remove sync to remote environments checkbox

### PR DESCRIPTION
**Pull request by andreluizmachado**
- Removed the checkbox during on delivery dialog (create/update).
- Removed Publish Actions From Add Platform Dialog
- Removed Delivery Synchronisation events

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -37,12 +37,10 @@
     'install' => array(
         'rdf' => array(
             __DIR__. '/model/ontology/platform.rdf',
-            __DIR__. '/model/ontology/taodelivery.rdf',
```
Comment (luisfn) [inline manifest.php:40]: Can't comment on correct line but you probably need to update the extension version

**Comment by luisfn** [comment]
You probably need to also update this too:

https://github.com/oat-sa/extension-tao-publishing/blob/feature/TAO-10138/remove-sync-to-remote-environments-checkbox/scripts/update/Updater.php

File: manifest.php (hunk 1 above)
Comment (andreluizmachado) [reply manifest.php:40]: We plan to do it once the new feature is done [here](https://github.com/oat-sa/extension-tao-publishing/pull/63/files)

Comment (Skyld) [reply manifest.php:40]: I think it's not necessary since the target branch is an integration branch

**Pull request merged**